src/main/java/nextstep/ladder/controller/LadderCame.java

```diff
@@ -0,0 +1,26 @@
+package nextstep.ladder.controller;
+
+import nextstep.ladder.domain.Height;
+import nextstep.ladder.domain.Ladder;
+import nextstep.ladder.domain.Participants;
+import nextstep.ladder.util.StringParser;
+
+import java.util.List;
+
+import static nextstep.ladder.ui.InputView.inputLadderHeight;
+import static nextstep.ladder.ui.InputView.personName;
+import static nextstep.ladder.ui.OutputView.printLadder;
+import static nextstep.ladder.ui.OutputView.printParticipants;
+
+public class LadderCame {
+
+    public static void main(String[] args) {
+        List<String> persons = StringParser.split(personName());
+        Participants participatns = new Participants(persons);
```

Comment:
```suggestion
        Names participatns = new Names(persons);
```

저는 `Names`로 일반화하고 
불려지는곳에서의 이름으로 역할을 설명할것 같아요 😄